packages/wdio-devtools-service/src/utils.js

```diff
@@ -70,3 +72,59 @@ export async function readIOStream (cdp, stream) {
 export function sumByKey (list, key) {
     return list.map((data) => data[key]).reduce((acc, val) => acc + val, 0)
 }
+
+/**
+ * check if url is supported for tracing
+ * @param  {String}  url to check for
+ * @return {Boolean}     true if url was opened by user
+ */
+export function isSupportedUrl (url) {
+    return IGNORED_URLS.filter((ignoredUrl) => url.startsWith(ignoredUrl)).length === 0
+}
+
+/**
+ * Approximates the Gauss error function, the probability that a random variable
+ * from the standard normal distribution lies within [-x, x]. Moved from
+ * traceviewer.b.math.erf, based on Abramowitz and Stegun, formula 7.1.26.
+ * @param {number} x
+ * @return {number}
+ */
+function internalErf_ (x) {
+    // erf(-x) = -erf(x);
+    const sign = x < 0 ? -1 : 1
+    x = Math.abs(x)
+
+    const a1 = 0.254829592
+    const a2 = -0.284496736
+    const a3 = 1.421413741
+    const a4 = -1.453152027
+    const a5 = 1.061405429
+    const p = 0.3275911
+    const t = 1 / (1 + p * x)
+    const y = t * (a1 + t * (a2 + t * (a3 + t * (a4 + t * a5))))
+    return sign * (1 - y * Math.exp(-x * x))
+}
+
+/**
+ * Creates a log-normal distribution and finds the complementary
+ * quantile (1-percentile) of that distribution at value. All
+ * arguments should be in the same units (e.g. milliseconds).
+ *
+ * @param {number} median
+ * @param {number} falloff
+ * @param {number} value
+ * @return The complement of the quantile at value.
+ * @customfunction
+ */
+export function quantileAtValue (median, falloff, value) {
+    const location = Math.log(median)
+
+    // The "falloff" value specified the location of the smaller of the positive
+    // roots of the third derivative of the log-normal CDF. Calculate the shape
+    // parameter in terms of that value and the median.
+    const logRatio = Math.log(falloff / median)
+    const shape = Math.sqrt(1 - 3 * logRatio - Math.sqrt((logRatio - 3) * (logRatio - 3) - 8)) / 2
```

Comment:
I pretty much copy&pasted the code from the google sheet, did you test this?

Reply:
yeah, just copied internalErf_ and quantileAtValue functions to browser console and called quantileAtValue like this:
`quantileAtValue(0, 1, 2)` -> NaN // median === 0
`quantileAtValue(1, 1, 1)` -> NaN // falloff => value
`quantileAtValue(0, 1, 2)` -> NaN // falloff >= value

However I'm not sure if it make sense to do something here because I don't know what arguments can be received by `quantileAtValue`

Reply:
me neither .. as I said, it is copy pasted from the google spreadsheet and I don't think it will cause issues, at least I haven't experienced any